devserverconfig.json

```diff
@@ -2,26 +2,10 @@
     "port": 3001,
 
     "allowProxyFor" : [
-        "nicta.com.au",
-        "gov.au",
-        "csiro.au",
-        "arcgis.com",
-        "argo.jcommops.org",
-        "www.abc.net.au",
-        "geoserver.aurin.org.au",
-        "mapsengine.google.com",
-        "s3-ap-southeast-2.amazonaws.com",
-        "adelaidecitycouncil.com",
-        "www.dptiapps.com.au",
-        "geoserver-123.aodn.org.au",
-        "geoserver.imos.org.au",
-        "nci.org.au",
-        "static.nationalmap.nicta.com.au",
-        "githubusercontent.com",
-        "gov",
-        "gov.uk",
-        "gov.nz",
-        "sample.aero3dpro.com.au"
-    ]
+        "dev-csvs.sa-catapult.co.uk"
```

Comment:
Why do we need this file? Are those values not overridden by the config map?

Reply:
this file is for dev and testing purposes only